### PR TITLE
DOC-537 Update Redis Install Steps for IC and Drupal 9

### PR DESCRIPTION
**Pull request by whitneymeredith**
Closes # [DOC-537](https://getpantheon.atlassian.net/jira/software/projects/DOC/boards/186?selectedIssue=DOC-537)

## Summary

**[Object Cache](https://pantheon.io/docs/object-cache#enable-object-cache)** - Updated steps for installing Redis in IC and Drupal 9 environments to clarify that installation should take place with Composer, not Drush.

## Effect

The following changes are already committed:

*
*

## Remaining Work and Prerequisites

The following changes still need to be completed:

- [ ] List any outstanding work here

### Dependencies and Timing

- [ ] Other prerequisites that must be completed before merging this PR

**Release**:
- [ ] When ready
- [ ] After date: $DATE

--------------------------------------------------

## Post Launch

**Do not remove** - To be completed by the docs team upon merge:

- [ ] Redirect `/docs/old-path/` => `/docs/new-path/` (if applicable)
- [ ] Include/exclude pages ^ respectively within docs search service provider (if applicable)
- [ ] For Heroes - add a props post to the [discussion board](https://discuss.pantheon.io/c/pantheon-platform/documentation/17).
- [ ] Remove from the [project board](https://github.com/pantheon-systems/documentation/projects/14)

### CODE REVIEW

**Comment by pantheondocs** [comment]
Lighthouse Audit Results:

| Description | [Live](https://pantheon.io/docs) | [pr-7280](https://pr-7280--pantheon-docs.my.pantheonfrontend.website/docs) | Difference |
| --- | --- | --- | --- |
| Performance Score | `48` | `46` | `-2` |
| Best Practice Score | `86` | `86` | `0` |
| SEO Score | `96` | `96` | `0` |
| Accessibility Score | `98` | `98` | `0` |

Lighthouse Audit Reports:

- [`pr-7280` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/36324/artifacts/0//lighthouse_data/lighthouse-audit-dev.html)
- [`Pantheon Docs` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/36324/artifacts/0//lighthouse_data/lighthouse-audit-reference.html)


[Preview Environment](https://pr-7280--pantheon-docs.my.pantheonfrontend.website/docs)

**Comment by whitneymeredith** [comment]
[Preview](https://pr-7280--pantheon-docs.my.pantheonfrontend.website/docs/object-cache)

File: source/content/object-cache.md [hunk 1]
```diff
@@ -128,19 +128,99 @@ All plans except for the Basic plan can use Object Cache. Sandbox site plans can
...
+  ```sql
+  TRUNCATE TABLE `<tablename>`;
+  ```
+
+</Accordion>
+
+</Tab>
+
+
 <Tab title="Drupal 8" id="d8-install">
```
Comment (whitneymeredith) [inline source/content/object-cache.md:209]: @greg-1-anderson Can you confirm that the steps here are correct?

File: source/content/object-cache.md [hunk 2]
```diff
@@ -178,33 +258,33 @@ All plans except for the Basic plan can use Object Cache. Sandbox site plans can
...
-```
+  ```sql
+  TRUNCATE TABLE `<tablename>`;
+  ```
 
 </Accordion>
 
 </Tab>
 
 <Tab title="Drupal 9 / Composer-managed" id="d9-install">
```
Comment (whitneymeredith) [inline source/content/object-cache.md:285]: @greg-1-anderson Can you review the steps here as well?

File: source/content/object-cache.md [hunk 3]
```diff
@@ -128,19 +128,99 @@ All plans except for the Basic plan can use Object Cache. Sandbox site plans can
...
+1. Download and activate the [Redis](https://www.drupal.org/project/redis) module from Drupal.org.
 
   <Alert title="Note" type="info">
 
   You **must** activate the module before proceeding.
 
   </Alert>
 
-  You can install and enable the module from the command line using [Terminus](/terminus):
+1. Run `composer require drupal/redis` in the local command line to install Redis.
```
Comment (greg-1-anderson) [inline source/content/object-cache.md:221]: After installing locally, it is necessary to commit and push the change to Pantheon:
```
git commit --am "Add drupal/redis dependency"
git push origin master
```
This must be done before running the `terminus drush` command below. The Drush command might fail if the build started by the `git push` has not finished yet.

Comment (greg-1-anderson) [reply source/content/object-cache.md:221]: That would go above the commit and push, of course.

**Review by greg-1-anderson — CHANGES_REQUESTED** [review]
Mostly good, just a couple of suggestions.

**Comment by wordsmither** [comment]
lgtm

**Pull request merged**